src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -99,6 +99,8 @@ export const activityFeedInlineError: Errors = {
     },
 };
 
+const MARK_NAME_DATA_LOADING = `${ORIGIN_ACTIVITY_SIDEBAR}_data_loading`;
```

Comment:
should `data_loading` also be a constant, like `EVENT_DATA_LOADING`?

Reply:
I didn't because the constant is an actual event name being submitted to the analytics store whereas `data_loading` is just a suffix for a marker to measure performance